cetmix_tower_yaml/static/description/index.html

```diff
@@ -396,34 +401,54 @@ <h1 class="title">Cetmix Tower YAML</h1>
 </ul>
 </div>
 <div class="section" id="configuration">
-<h1><a class="toc-backref" href="#toc-entry-1">Configuration</a></h1>
+<h2><a class="toc-backref" href="#toc-entry-1">Configuration</a></h2>
 <p>Please refer to the <a class="reference external" href="https://cetmix.com/tower">official
 documentation</a> for detailed configuration
 instructions.</p>
 </div>
 <div class="section" id="usage">
-<h1><a class="toc-backref" href="#toc-entry-2">Usage</a></h1>
+<h2><a class="toc-backref" href="#toc-entry-2">Usage</a></h2>
 <p>Please refer to the <a class="reference external" href="https://cetmix.com/tower">official
 documentation</a> for detailed usage
 instructions.</p>
 </div>
 <div class="section" id="changelog">
-<h1><a class="toc-backref" href="#toc-entry-3">Changelog</a></h1>
+<h2><a class="toc-backref" href="#toc-entry-3">Changelog</a></h2>
 <div class="section" id="section-1">
+<<<<<<< HEAD
 <h2><a class="toc-backref" href="#toc-entry-4">17.0.1.1.1 (2025-07-04)</a></h2>
+=======
+<h3><a class="toc-backref" href="#toc-entry-4">16.0.1.1.1 (2025-06-23)</a></h3>
+<ul class="simple">
+<li>Features: YAML code optimisation (4728)</li>
+</ul>
+</div>
+<div class="section" id="section-2">
+<h3><a class="toc-backref" href="#toc-entry-5">16.0.1.1.0 (2025-06-20)</a></h3>
+>>>>>>> f8b8067 ([FIX]  cetmix_tower_yaml: bypass SSH check)
 <ul class="simple">
 <li>Features: Export/import scheduled tasks to/from YAML. (4650)</li>
 </ul>
 </div>
+<<<<<<< HEAD
 <div class="section" id="section-2">
 <h2><a class="toc-backref" href="#toc-entry-5">17.0.1.1.0 (2025-06-30)</a></h2>
+=======
+<div class="section" id="section-3">
+<h3><a class="toc-backref" href="#toc-entry-6">16.0.1.0.5 (2025-05-21)</a></h3>
+>>>>>>> f8b8067 ([FIX]  cetmix_tower_yaml: bypass SSH check)
 <ul class="simple">
 <li>Features: Export/import secret values related to Server. (4696)</li>
 <li>Features: YAML code optimisation (4728)</li>
 </ul>
 </div>
+<<<<<<< HEAD
 <div class="section" id="section-3">
 <h2><a class="toc-backref" href="#toc-entry-6">17.0.1.0.2 (2025-05-20)</a></h2>
+=======
+<div class="section" id="section-4">
+<h3><a class="toc-backref" href="#toc-entry-7">16.0.1.0.4 (2025-05-16)</a></h3>
+>>>>>>> f8b8067 ([FIX]  cetmix_tower_yaml: bypass SSH check)
```

Comment:
This is left from merge conflict resolve, you must resolve the conflicts properly